axelor-base/src/main/resources/domains/MetaTheme.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<domain-models xmlns="http://axelor.com/xml/ns/domain-models"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://axelor.com/xml/ns/domain-models http://axelor.com/xml/ns/domain-models/domain-models_7.4.xsd">
+
+  <module name="meta" package="com.axelor.meta.db"/>
+
+  <entity name="MetaTheme" table="META_THEME">
+
+    <integer name="logoModeSelect" title="Logo mode" selection="meta.theme.logo.mode.select"
+      default="1"/>
```

Comment:
can't we use a real enumeration instead?

Reply:
I can add new Enum like ThemeLogoMode 
What do you think ?

Reply:
`ThemeLogoMode` enum added with 3 values 
`NONE` to manage empty case